zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -140,6 +140,35 @@ <h1>Edit upload</h1>
       <h1>New version</h1>
       {%- else %}
       <h1>New upload</h1>
+      {% if record.files %}
+      {%- if current_user_is_admin() %}
```

Comment:
```suggestion
      {# Note: we need an already initialized deposit #}
      {% if record.files and current_user_is_admin() %}
```

More compact, less Jinja blocks nesting, and a comment for some clarity :)

(Remember to remove the `endif` later on. Also test it locally since I can't even remember if that's what Jinja comments looks like :) )